### ISSUE TITLE
Webm format not readable by Forefox, VideoLAN

### ISSUE THREAD

**Issue by anjseattle** [drawpile/Drawpile#842]
Hi!
I am using the 64-bit 2.1.15 version with Windows 10 on a Surface Pro. Pen pressure works great, video recording & playback works within Drawpile although a bit touchy.

I have exported webm format - it will not open in Firefox, nor in VideoLAN. Error messages for both. Both Firefox & VideoLAN should support webm format, so I suspect it is the file.

**Comment by Wade821** [comment]
Closing issue, duplicate of #248. Please comment there for further discussion. 

_**As a side note**_, if you do have Discord, you can post suggestions there by joining it and even discussing the viability of such suggestions. The invite link is found on Drawpile.net in the bottom left, or you can use this one ( https://discord.gg/M3yyMpC ). I'm a moderator there and I go by Bluestrings. If you do join, reference this suggestion and start a conversation! So we can maybe review in more detail or close it out.

**Comment by Wade821** [comment]
This is more of the same issue though? It's either that we close this or the old.

**Comment by callaa** [comment]
I don't think this is a duplicate of 248, since webm export didn't exist back then. I believe the problem is related to the color space. Currently, the video is exported in RGB color space, which doesn't seem to be too well supported.

**Comment by Wade821** [comment]
Ah. I was going more with video export not working than I was specifics to the format. Should the old one be closed then?

**Comment by askmeaboutlo0m** [comment]
(I've taken over development on Drawpile, so I'm going through these tickets and organizing them.)

Video export has been replaced since then, so I don't think this is relevant anymore.
